SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/CylinderModel.cpp

```diff
@@ -42,13 +36,7 @@ int RigidCylinderCollisionModelClass = core::RegisterObject("Collision model whi
         .addAlias("CylinderModel")
```

Comment:
didn't even know we had that in SOFA ....

Reply:
😙